src/Storage/Storage.Management/help/New-AzStorageBlobInventoryPolicyRule.md

```diff
@@ -16,13 +16,14 @@ Creates a blob inventory policy rule object, which can be used in Set-AzStorageB
 ```
 New-AzStorageBlobInventoryPolicyRule [-Name] <String> [-Disabled] -Destination <String> -Format <String>
  -Schedule <String> -BlobSchemaField <String[]> -BlobType <String[]> [-PrefixMatch <String[]>]
- [-IncludeSnapshot] [-IncludeBlobVersion] [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+ [-ExcludePrefix <String[]>] [-IncludeSnapshot] [-IncludeBlobVersion] [-IncludeDeleted]
```

Comment:
Better to update example #1, #2 in https://github.com/Azure/azure-powershell/blob/ff4875a75c05c02bbc47f7cb13ea6e8937179ed6/src/Storage/Storage.Management/help/Set-AzStorageBlobInventoryPolicy.md

Reply:
Done